src/syscalls/platform_mock.rs

```diff
@@ -9,18 +16,30 @@ pub unsafe fn subscribe(
     unimplemented()
 }
 
+/// Send a command to the tock kernel
+/// # Safety
+/// This function usually involves assembly calls which are unsafe.
```

Comment:
I don't understand this justification: sure, using assembly in the implementation is unsafe, but that shouldn't affect callers. What about command() is unsafe to callers?

Reply:
See https://github.com/tock/libtock-rs/pull/126#discussion_r363100800

Reply:
Ah, got it. I wish GitHub would indicate that a file had previously been commented on when I am reviewing it myself :-(